tools/autograd/derivatives.yaml

```diff
@@ -297,6 +297,9 @@
 - name: _ctc_loss(Tensor log_probs, Tensor targets, int[] input_lengths, int[] target_lengths, int blank=0, bool zero_infinity=False) -> (Tensor, Tensor)
   log_probs: _ctc_loss_backward(grad, log_probs, targets, input_lengths, target_lengths, result0, result1, blank, zero_infinity)
 
+- name: deg2rad(Tensor self) -> Tensor
+  self: grad
```

Comment:
This one is missing a multiplier right?

Reply:
I wanted to know how to make that constant available in `derivatives.yaml`. Now that you've told me how, I'll make that change. Thanks!